src/consensus/merkle.cpp

```diff
@@ -410,9 +410,8 @@ void MerkleProof::clear() noexcept
 
 void swap(MerkleProof& lhs, MerkleProof& rhs)
 {
-    using std::swap;
```

Comment:
As commented via email, this may differ from what bitcoin generally does, but it is in fact the correct idiom for recent versions of C++:

http://en.cppreference.com/w/cpp/language/adl